From e8c108ca9f11a382742f212f5b42a02536b3d40f Mon Sep 17 00:00:00 2001
From: Philippe De Swert <philippedeswert@gmail.com>
Date: Wed, 17 Sep 2014 00:27:16 +0300
Subject: [PATCH] journal: Do not count on the compiler initializing found_last
 to false

There is a very unlikely case where this can happen since gcc usually
does the sane thing. But let's make sure found_last is initialized anyway.

Fixes: CID#996386
---
 src/journal/journal-verify.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/journal/journal-verify.c b/src/journal/journal-verify.c
index 6c8ca8c268..b4e8f73c41 100644
--- a/src/journal/journal-verify.c
+++ b/src/journal/journal-verify.c
@@ -804,7 +804,7 @@ int journal_file_verify(
         usec_t last_usec = 0;
         int data_fd = -1, entry_fd = -1, entry_array_fd = -1;
         unsigned i;
-        bool found_last;
+        bool found_last = false;
 #ifdef HAVE_GCRYPT
         uint64_t last_tag = 0;
 #endif
